### PR TITLE
feat(docs): new Entity & relationships map features

### PR DESCRIPTION
**Pull request by ghost**
Update screenshots based on UI changes and consolidated low-traffic pages.

Jira: NR-172577

### CODE REVIEW

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  - /docs/new-relic-one/use-new-relic-one/ui-data/service-maps-new-relic-one
+  - /docs/new-relic-one/use-new-relic-one/ui-data/service-maps/how-use-service-maps
+  - /docs/new-relic-one/ui-data/service-maps/how-use-service-maps
+  - /docs/new-relic-solutions/new-relic-one/ui-data/service-maps/how-use-service-maps
+  - /docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps
+---
+
+import mapsSolutionsServiceMapsWithOmnimap from 'images/mapsSolutions_screenshot-full_service-maps-with-omnimap.webp'
+
+Service maps are visual, dynamic representations of your architecture. Maps automatically show your app's connections and dependencies, including applications, databases, hosts, servers, and out-of-process services.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:49]: Note to self: @amezquita slack him

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+### Filter entities [#filter-map]
+
+You can modify the service map based on things like:
+
+* **Health status**: Remove the colored health outline around affected entities.
+* **Entity types**: Hide specific entities.
+* **Relationship depth**:
+  * **Depth 1**: (Default) Show all entities.
+  * **Depth 2**: Show degraded entities only.
+  * **Depth 3**: Show degraded entities only.
```
Comment (ally-sassman) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:112]: @joandiazcapell What's the difference between depth 2 vs. 3? What's an example of what you'd see if you went from depth 1 to depth 2 to depth 3?

Comment (joandiazcapell) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:112]: By default, you are in relationship depth 1. This means you see all entities that have a direct relationship with the main entity. If you go to depth 2, you will go one hop ahead on relationships, you will also see direct relationships of these entities related to the main entity, and if you move to depth 3, the same, one hop more. The trick is, on depths 2 and 3 you only see entities with alerts. If we pain all entities you could end up with a huge unreadable map full of entities.
In this image, the main entity "account-provisional" has some relationships, if you move to depth 2 you will see all these new red entities on the top (inside a red box) that don't have a direct relationship from the main entity, they are two hops away from the main entity, they have the direct relationships with "Service - APm" which is not the main entity (depth 2)
Screenshot 2023-
11-07 at 09 45 29

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 3]
```diff
@@ -0,0 +1,165 @@
+---
+title: "Service map: Visualize your entities"
+tags:
+  - Understand dependencies
+  - Understand system dependencies
+  - Service maps
+metaDescription: 'New Relic''s service maps are visual, customizable representations of your application architecture.'
+redirects:
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:8]: These all look right to me! I didn't verify each individual redirect, but as long as you did the following (which it seems you did) it's good!

- Moved all redirects found in deleted docs to this doc
- Added deleted docs to this redirect list

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+import solutionsMapEntity from 'images/solutions_screenshot-full_map-entity-example.webp'
+
+import solutionsFilterServiceMap from 'images/solutions_screenshot-full_filter-service-map.gif'
+
+import solutionsServiceMapsRightPane from 'images/solutions_screenshot-full_service-maps-right-pane.gif'
+
+import solutionsMapfromthisentityexample from 'images/solutions_screenshot-crop_mapfromthisentityexample.webp'
+
+Service map is a visual representations of your entire architecture from front-end to back-end, providing a comprehensive overview of the interconnected components that make up your system. This map dynamically displays the relationships between your applications, databases, hosts, servers, and out-of-process services (called [web externals]([#externals])).
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:71]: The `#externals` link doesn't point to an existing header/doc

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+  src={SolutionsServiceMaps}
+/>
+
+<figcaption>
+  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM > (choose an entity) > Service map**: Service map shows your dependencies and how they're performing.
+</figcaption>
+
+## Use service map [#using-service-maps]
+
+<Callout variant="tip">
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:85]: Starting the meaty part of the doc with a tip that doesn't _really_ matter breaks the flow a bit too much imo. I'd try to find another location to place this callout.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+  Service map works with [distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) to connect relationships between entities. Service map is still functional if you haven't enabled distributed tracing, but we recommend having distributed tracing enabled for all agents. This ensures a more consistent experience while using service maps.
+  </Callout>
+
+To access service map: 
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, then select either **APM & services**, **Mobile**, or **Browser**.
+2. Select an entity.
+3. On the left-hand side, click **Service map**.
+
+### Map from one entity [#map-from-one-entity]
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:94]: This heading is a bit confusing. Maybe `Map from specific entity`? You can ignore this comment if you disagree - this one's small.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+Hover your cursor over an entity and click "Map from this entity" to view how entities are related from that entity's perspective. The service map will update accordingly.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:96]:
```suggestion
Hover your cursor over an entity and click **Map from this entity** to view how entities are related from that entity's perspective. The service map will update accordingly.
```

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+<img
+    title="Example of map from this entity"
+    alt="A screenshot of how service map updates after mapping from a specific entity."
+    src={solutionsMapfromthisentityexample}
+/>
+
+### Filter entities [#filter-map]
+
+You can filter the service map based:
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:113]: Does this need a procedure/quick explanation of _how_ to filter?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+  * **Depth 2**: Focuses on degraded entities (with alerts) that are two hops away from the main entity. Entities with direct connections to the main entity are not included in this view.
+  * **Depth 3**: Similar to depth 2, depth 3 concentrates on degraded entities with alerts but extends the search to entities that are three hops away from the main entity. Direct connections to the main entity remain excluded.
+
+  <Callout variant="tip">
+  The purpose of limiting the display of entities to degraded ones in depths 2 and 3 is to prevent the service map from becoming cluttered and overwhelming. As you increase the relationship depth, you are going further into the network of entities, but you are only seeing entities that are experiencing issues. This can help you to focus on the most critical problems in your system.
+    </Callout>
+ * **Timewarp**: Explore how entity health changed in the last three hours. 
+
+
+<img
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:128]: I'd move this gif to the top of the h3

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+* **Entity types**: Hide specific entities from the map.
+* **Relationship depth**:
+  * **Depth 1**: (Default) Displays all entities directly connected to the main entity, regardless of their health status.
+  * **Depth 2**: Focuses on degraded entities (with alerts) that are two hops away from the main entity. Entities with direct connections to the main entity are not included in this view.
+  * **Depth 3**: Similar to depth 2, depth 3 concentrates on degraded entities with alerts but extends the search to entities that are three hops away from the main entity. Direct connections to the main entity remain excluded.
+
+  <Callout variant="tip">
+  The purpose of limiting the display of entities to degraded ones in depths 2 and 3 is to prevent the service map from becoming cluttered and overwhelming. As you increase the relationship depth, you are going further into the network of entities, but you are only seeing entities that are experiencing issues. This can help you to focus on the most critical problems in your system.
+    </Callout>
+ * **Timewarp**: Explore how entity health changed in the last three hours.
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:125]: Move this timewarp section above the depth sections so the callout ends the list

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+ * **Timewarp**: Explore how entity health changed in the last three hours. 
+
+
+<img
+    title="Search, filter, and explore your entities"
+    alt="A GIF of how service map updates when filtering the health status, timewarp, entity type, and relationship depth."
+    src={solutionsFilterServiceMap}
+/>
+
+<Callout variant="tip" title="Externals and databases in maps">
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:134]: This is _very_ useful info, but i'm not sure a callout is the best way to present it. It almost seems like it should be it's own section of the doc. Even if you do decide to keep it as a callout, I'm not sure this is the place for it to exist - unless I'm missing how this relates to filtering.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+
+In the New Relic UI, your out-of-process services are referred to as [web external](/docs/using-new-relic/welcome-new-relic/get-started/glossary#web-external) or background external data. Externals and databases have slightly different features in service maps than other entity types:
+
+* Unlike other entities that appear in service maps, **externals** are aggregates. Clicking on an external service in the map shows you the list of all the external services that are rolled up into the one external entity. This is to reduce map clutter, as some entities can have dozens of externals being reported.
+* Due to their agentless nature, databases cannot have alerts set for them; only service calls made to the database are reported to New Relic.
+  </Callout>
+
+### View entity performance metrics [#performance-data]
```
Comment (bradleycamacho) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:142]: Maybe a quick list of relevant performance metrics?

Comment (bradleycamacho) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/service-maps.mdx:142]: Take this suggestion as optional

**Pull request merged**